libopencm3.nix

```diff
@@ -11,12 +11,12 @@
 
 stdenvNoCC.mkDerivation rec {
   pname = "libopencm3";
-  version = "ec5aeba354ec273782e4441440fe9000b1c965e3";
+  version = "228f28fa66b672535663aa9aa4d12ede8b1a8fc5";
   src = fetchFromGitHub {
     owner = "libopencm3";
     repo = pname;
-    rev = "ec5aeba354ec273782e4441440fe9000b1c965e3";
-    sha256 = "sha256-bgoMhOhBJZwPTa9gUH0vPSGZknDrb2mJZuFlCWNivYU=";
+    rev = "228f28fa66b672535663aa9aa4d12ede8b1a8fc5";
+    sha256 = "sha256-eqjOtvH+FwRaWqVSxUyK/gXtveEipLsyjd4jLzXbzbw=";
```

Comment:
Why do you need to update libopencm3? Is this needed for adding the support for this board? If not then this should be done as a separate PR so we have an easier time to trace back problems should they occur later.

Reply:
revert in https://github.com/pq-code-package/mlkem-c-embedded/commit/ee07cad0dd81afa57ed849349e73ac60da0d3f5d